profile/README.md

```diff
@@ -1,15 +1,41 @@
 # Think big
 
-![](https://archbee-image-uploads.s3.amazonaws.com/CGGF0VLgJQypTJbZ1FbQy/aoF2F0Tr5bORp7wnTzRec_github-org-page-banner.png)
+![](https://archbee-image-uploads.s3.amazonaws.com/CGGF0VLgJQypTJbZ1FbQy/aoF2F0Tr5bORp7wnTzRec_github-org-page-banner.png)  
+ 
+&nbsp;
+&nbsp;
+&nbsp;
+
+Voltage enables your team to create enterprise-grade experiences. By leveraging Voltage you receive the fastest and most scalable Lightning solution that meets the highest security and privacy standards.
+
+With modern modular tools and processes, you can build, deploy, and scale faster than ever.
+Voltage brings a suite of lightning node and data services to the market that elevates your value proposition through Lightning Network architecture.
+
+You are here to either setup enterprise grade Bitcoin infrastructure in less than two minutes, or seek further information on Voltage, so let’s get you where you need to be.
+
+&nbsp;
+
+[![X](Xiconx2.png)][1] &nbsp; [![linkedin](Linkediniconx2.png)][2] &nbsp; [![discord](Discordiconx2.png)][3]
```

Comment:
<img width="198" alt="Screenshot 2024-07-31 at 11 24 22 AM" src="https://github.com/user-attachments/assets/7e052a34-1e1f-4fb1-9657-5befc881016e">

When I preview in VScode, these icons looks a little offsized. Some are bigger than others. The "X" is going to completely disappear in dark mode, which is what a lot of devs like anyways (or at least a vocal minority that has to be appeased in every single election).

What if it was like 3 evenly sized titles? Each could have a white background with rounded corners, and a black single-color icon in the middle. Or whatever you think looks good, but point is, they would all be consistently sized and visible in light and dark mode.